hrm-domain/hrm-core/permissions/canPerformActionOnObject.ts

```diff
@@ -64,6 +64,12 @@ export const canPerformActionsOnObject = async <T extends TargetKind>({
 
         return newOk({ data: canPerform });
       }
+      case 'profile': {
+        throw new Error('Not implemented');
+      }
+      case 'profileSection': {
+        throw new Error('Not implemented');
+      }
```

Comment:
Do we want to implement this? This endpoint was initially implemented to check on permissions before uploading/downloading documents, and ended up serving all possible actions for a given object. I could extend now, or defer for when this is used - if ever.